<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Decoded text preview window</string>
  </property>
  <widget class="QPlainTextEdit" name="text_tb">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>811</width>
     <height>581</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Liberation Mono</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Decoded Text:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
